core/src/main/java/org/jdbi/v3/core/Handles.java

```diff
@@ -50,8 +50,9 @@ public boolean isForceEndTransactions() {
      * @param forceEndTransactions whether to enforce transaction termination
      *                             discipline.
      */
-    public void setForceEndTransactions(boolean forceEndTransactions) {
+    public Handles setForceEndTransactions(boolean forceEndTransactions) {
```

Comment:
This is a backwards incompatible change and cannot be merged.

Reply:
https://docs.oracle.com/javase/specs/jls/se7/html/jls-13.html#jls-13.4.15

Reply:
I had such a feeling...